src/ert/scheduler/openpbs_driver.py

```diff
@@ -66,6 +69,52 @@ def _resource_string(self) -> str:
             resource_specifiers += ["mem=" + self._memory_per_job]
         return ":".join(resource_specifiers)
 
+    async def _execute_with_retry(
+        self,
+        cmd_with_args: List[str],
+        fail_exit_codes: List[int],
+    ) -> Tuple[bool, str]:
+        error_message: Optional[str] = None
+
+        for _ in range(self._num_retries):
+            try:
+                process = await asyncio.create_subprocess_exec(
+                    *cmd_with_args,
+                    stdout=asyncio.subprocess.PIPE,
+                    stderr=asyncio.subprocess.PIPE,
+                )
+                stdout, stderr = await process.communicate()
+
+                if process.returncode == 0:
+                    return True, stdout.decode().strip()
+                elif process.returncode in fail_exit_codes:
+                    # expected exit code when failed command
+                    error_message = stderr.decode().strip()
+                else:
+                    error_message = stderr.decode().strip()
+                    error_message = (
```

Comment:
suggest not to overwrite the variable, use a different variable name on line 94, or inline line 94 in line 95.